### PR TITLE
Adding new autocomplete values to input fields

### PR DESCRIPTION
**Pull request by tipue-dev**
Google Chrome ignores the autocomplete value "off" and fills the password fields in AdminMailAccount (as mentioned here https://github.com/znuny/Znuny/pull/64#issuecomment-825732068) and AdminSMIME.
The input fields for the TwoFactorToken don't have any special autocomplete attribute (as mentioned here https://github.com/znuny/Znuny/pull/64#issuecomment-825742881).

## Proposed change

Add ``autocomplete="new-password"`` to the password input in AdminMailAccount and AdminSMIME.
Add ``autocomplete="one-time-code`` to the 2 factor token fields in Login.tt and CustomerLogin.tt and their preferences.

These seem fitting to me. 

1 - 💎 code quality

## Additional information

- This PR is related to PR: https://github.com/znuny/Znuny/pull/64

I did not add the commits to the related PR, which is mine too, because i can't (to my knowledge) add commits
to a PR from a repository i don't have access to. But please correct me if there is a way.

## Checklist

- [x] The code change is tested and works locally.(❗)
- [x] There is no commented out code in this PR.(❕)
- [ ] You improved or added new unit tests.(❕)
- [x] Local ZnunyCodePolicy run passes successfully.(❕)
- [x] Local unit tests pass.(❕)
- [x] GitHub workflow ZnunyCodePolicy passes.(❗)
- [x] GitHub workflow unit tests pass.(❗)

### CODE REVIEW

**Comment by rkaldung** [comment]
Hi, during QA I saw that the password manager pops up when adding a new agent or  customer user. Any chance to prevent this? Make no sense for me to trigger the password manager when adding one of them.

**Comment by tipue-dev** [comment]
Hi, as far as i can see, there is no really good solution. We changed the attribute for the password field from ```autocomplete="off"``` to ```autocomplete="new-password"``` in PR https://github.com/znuny/Znuny/pull/64 because some browsers ignore the "off" value and auto fill username and password [developer.mozilla.org 1](https://developer.mozilla.org/en-US/docs/Web/Security/Securing_your_site/Turning_off_form_autocompletion#preventing_autofilling_with_autocompletenew-password). "new-password" suppresses this, but this is still seen as user credentials for the browser to save. This must have triggered your password manager.

Another possibility would be to change the attributes value to "one-time-code" [developer.mozilla.org 2](https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/autocomplete). This prevents chrome from saving the password for a user, maybe this also helps with external password managers.

You'll still get suggestions from previously used passwords in both cases.

The cleanest solution would be, if the browsers don't recognize the password field at all, but chrome also uses context to find out if this is a password field. It even checks labels to see, if the label says "password" or something similar (_I found some references the last time, but a short google search didn't help this time_).
This is also the reason, why chrome has a password layover over the authentication dropdown for basic auth in the properties action for rest webservices.

**Comment by rkaldung** [comment]
@tipue-dev Thanks for clarifying this. Maybe we address this later

File: Kernel/Output/HTML/Preferences/Password.pm [hunk 1]
```diff
@@ -1,6 +1,7 @@
 # --
 # Copyright (C) 2001-2021 OTRS AG, https://otrs.com/
 # Copyright (C) 2021 Znuny GmbH, https://znuny.org/
+# Copyright (C) 2021 maxence business consulting GmbH, http://www.maxence.de
```
Comment (dennykorsukewitz) [inline Kernel/Output/HTML/Preferences/Password.pm]: We have changed our rules regarding copyright.
We therefore ask you to remove your newly added copyrights.

```
Znuny is an open fork of an existing software. 
So we have to respect the already given copyright of the original creators.

New files will be licensed using the AGPL Version 3. 
If you contribute code to the Znuny project you will get mentioned in the pull request 
incl. the commit, in CHANGES.md and in AUTHORS.md. 
We will not mention you in the file you provided or changed. 
Your work is highly appreciated and acknowledged 
but you contribute it to the project and your copyright will pass on to the fork itself.
```

https://github.com/znuny/Znuny/blob/rel-6_1-dev/CONTRIBUTING.md#licensing-copyright-and-credits


```suggestion
```

File: Kernel/Output/HTML/Templates/Standard/AdminMailAccount.tt [hunk 2]
```diff
@@ -1,6 +1,7 @@
 # --
 # Copyright (C) 2001-2021 OTRS AG, https://otrs.com/
 # Copyright (C) 2021 Znuny GmbH, https://znuny.org/
+# Copyright (C) 2021 maxence business consulting GmbH, http://www.maxence.de
```
Comment (dennykorsukewitz) [inline Kernel/Output/HTML/Templates/Standard/AdminMailAccount.tt]:
```suggestion
```

File: Kernel/Output/HTML/Templates/Standard/AdminSMIME.tt [hunk 3]
```diff
@@ -1,6 +1,7 @@
 # --
 # Copyright (C) 2001-2021 OTRS AG, https://otrs.com/
 # Copyright (C) 2021 Znuny GmbH, https://znuny.org/
+# Copyright (C) 2021 maxence business consulting GmbH, http://www.maxence.de
```
Comment (dennykorsukewitz) [inline Kernel/Output/HTML/Templates/Standard/AdminSMIME.tt]:
```suggestion
```

File: Kernel/Output/HTML/Templates/Standard/AgentPreferences.tt [hunk 4]
```diff
@@ -1,6 +1,7 @@
 # --
 # Copyright (C) 2001-2021 OTRS AG, https://otrs.com/
 # Copyright (C) 2021 Znuny GmbH, https://znuny.org/
+# Copyright (C) 2021 maxence business consulting GmbH, http://www.maxence.de
```
Comment (dennykorsukewitz) [inline Kernel/Output/HTML/Templates/Standard/AgentPreferences.tt]:
```suggestion
```

File: Kernel/Output/HTML/Templates/Standard/CustomerLogin.tt [hunk 5]
```diff
@@ -1,6 +1,7 @@
 # --
 # Copyright (C) 2001-2021 OTRS AG, https://otrs.com/
 # Copyright (C) 2021 Znuny GmbH, https://znuny.org/
+# Copyright (C) 2021 maxence business consulting GmbH, http://www.maxence.de
```
Comment (dennykorsukewitz) [inline Kernel/Output/HTML/Templates/Standard/CustomerLogin.tt]:
```suggestion
```

File: Kernel/Output/HTML/Templates/Standard/CustomerPreferences.tt [hunk 6]
```diff
@@ -1,6 +1,7 @@
 # --
 # Copyright (C) 2001-2021 OTRS AG, https://otrs.com/
 # Copyright (C) 2021 Znuny GmbH, https://znuny.org/
+# Copyright (C) 2021 maxence business consulting GmbH, http://www.maxence.de
```
Comment (dennykorsukewitz) [inline Kernel/Output/HTML/Templates/Standard/CustomerPreferences.tt]:
```suggestion
```

File: Kernel/Output/HTML/Templates/Standard/Login.tt [hunk 7]
```diff
@@ -1,6 +1,7 @@
 # --
 # Copyright (C) 2001-2021 OTRS AG, https://otrs.com/
 # Copyright (C) 2021 Znuny GmbH, https://znuny.org/
+# Copyright (C) 2021 maxence business consulting GmbH, http://www.maxence.de
```
Comment (dennykorsukewitz) [inline Kernel/Output/HTML/Templates/Standard/Login.tt]:
```suggestion
```

**Review by dennykorsukewitz — CHANGES_REQUESTED** [review]
Hi @tipue-dev,

Looks good so far.
We have changed our rules regarding copyright.
We therefore ask you to remove your newly added copyrights.

```
Znuny is an open fork of an existing software. 
So we have to respect the already given copyright of the original creators.

New files will be licensed using the AGPL Version 3. 
If you contribute code to the Znuny project you will get mentioned in the pull request 
incl. the commit, in CHANGES.md and in AUTHORS.md. 
We will not mention you in the file you provided or changed. 
Your work is highly appreciated and acknowledged 
but you contribute it to the project and your copyright will pass on to the fork itself.
```

https://github.com/znuny/Znuny/blob/rel-6_1-dev/CONTRIBUTING.md#licensing-copyright-and-credits

**Review by dennykorsukewitz — APPROVED** [review]

**Pull request merged**